source/Weapon.cpp

```diff
@@ -151,6 +151,25 @@ void Weapon::LoadWeapon(const DataNode &node)
 				turn = value;
 			else if(key == "inaccuracy")
 				inaccuracy = value;
```

Comment:
As you mention, this should have a block that handles the new child keywords rather than a separate Weapon-level keyword.

Reply:
Fixed in c747bc967bf39217463d1fea01ecb03dd4716a0c.